ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -209,6 +210,8 @@ public Response handleKsqlStatements(
       @Context final KsqlSecurityContext securityContext,
       final KsqlRequest request
   ) {
+    // Set masked sql statement if request is not from OldApiUtils.handleOldApiRequest
```

Comment:
This version doesn't look like it has an OldApiUtils file. I think that's fine if we have defensive `setMaskedSqlIfNeeded()` everywhere we use `request.maskedSql()`, but we should at least remove this comment to avoid confusion.

Reply:
Why do we change `setMaskedSqlIfNeeded` to `setMaskedSql`? I think we need to be defensive here

Reply:
Both are equally defensive, aren't they? Both will ensure that the masked sql is set after the method is run.

The reason I suggested switching to `setMaskedSql()` is because now that OldApiUtils is gone, I don't think masked sql will ever be set before this line is reached. So `setMaskedSqlIfNeeded()` is equivalent to `setMaskedSql()`.